android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java

```diff
@@ -42,6 +42,7 @@
 public class VPNStatusDetailActivity extends AbstractActivity {
 
     private final OnGetVPNClickListener onGetVPNClickListener;
+    private boolean isProtectedConnection;
```

Comment:
Still a beginner. Didn't know what else to do without changing the structure more. Any better solutions?
Did it so it connects to the public OnGetVPNClickListener